### PR TITLE
Add stream Settlements

### PR DESCRIPTION
**Pull request by luandy64**
# Description of change
This PR adds Settlements as a stream

# Manual QA steps
 - Ran the tap
 - Could not test pagination because I have no data
 
# Risks
 - Low
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Comment by luandy64** [comment]
This should be merged after #36 

And before we merge this, we need to finish the location chunking refactor on this branch

**Review by asaf-erlich — APPROVED** [review]
Looks good but wait for the other 2 PRs to merge first?

**Pull request merged**
